encrypted-media-respec.html

```diff
@@ -2024,9 +2024,21 @@
 
     <section>
       <h2><a>HTMLMediaElement</a> Extensions</h2>
+      
+      <!-- Put all the monkey-patching here -->
+      
       <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported.</p>
+      
+      <p>This specification adds two internal values to the <a def-id="htmlmediaelement"></a>: <var>waiting for key</var> which shall have a boolean value and <var>encrypted block queue</var> which shall be a queue of encrypted blocks awaiting decryption.</p>
```

Comment:
Potentially script could create a new MediaKeys for the key for which playback is blocked waiting on. Seems unlikely that would be desirable though. But if that was desired, we'd want to not be clearing the queue of encrypted blocks awaiting decryption in the setMediaKeys algorithm, so the new MediaKeys could decode where the old one left off.

Neither Firefox nor Chrome (last time I checked) supports changing the MediaKeys after playback has started. It would be a bit of a hassle for us to implement, so we're unlikely to bother to implement it unless a compelling use case comes up.